src/reaction_lammps_mupt/reaction_template_builder/lunar_client/molecule_template_preparation.py

```
@@ -1017,299 +1038,398 @@ def molecule_file_preparation(test_molecule_file, template_indexes):
+    # Equivalences section: write reactant_index  product_index pairs.
     map_file += "\nEquivalences\n\n"
     for key, value in reactant_to_product.items():
-        map_file += f"{(key + 1):<5} {value + 1}\n"  # 1-based indexing
+        # key, value are 0-based internally; write as 1-based
+        map_file += f"{(key + 1):<5} {value + 1}\n"
 
-    # If deleted IDs were provided, append a #DeletedIDs block. Each deleted ID is
-    # written as a commented line (prefixed with '#') and converted to 1-based indexing.
+    # If delete IDs are present, output them prefixed with '#' on their own lines.
     if delete_ids:
         map_file += f"\n#DeleteIDs\n\n"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)